apps/myjobs/config/storage.yml

```diff
@@ -0,0 +1,34 @@
+test:
+  service: Disk
+  root: <%= Rails.root.join("tmp/storage") %>
+
+local:
+  service: Disk
+  root: <%= Rails.root.join("storage") %>
```

Comment:
This app does not use ActiveStorage so can we safely delete this config or is this just ignored? Similar question to the ActionCable config.

Reply:
Removed the file, the application doesn't load active_storage now.